Projects/Team1/Prystopchuk/Converter.cs

```diff
@@ -0,0 +1,69 @@
+﻿/*Method that converts a given number 
+ * from decimal to other base-N systems (N<=36)
+ * Return type: string*/
+
+namespace TEMP
```

Comment:
is it a correct namespase?

Reply:
oops, nope, I've forgotten to change it (I was writing this code in another project (It may be stupid, I know, but i cant get out of the habit to code in a temprorary project))